packages/variable-editor/src/components/detail/EmptyDetail.css

```diff
@@ -0,0 +1,16 @@
+.empty-detail {
+  flex: auto;
+  align-items: center;
+  justify-content: center;
```

Comment:
these two are also available as properties on the flex component, if you like :)